NEWS

```diff
@@ -1,3 +1,210 @@
+   - The new API adds the ability to transform spatiotemporal
+     coordinates.
+   - Functions in the new API uses the "proj_" namespace.
+   - Datatypes in the new API uses the "PJ_" namespace, with a few
+     historic exceptions such as XY, XYZ, LP and LPZ.
+
+ o Introduced the concept of "transformation pipelines" that makes it
+   possible to do complex geodetic transformations of spatiotemporal
+   coordinates in PROJ by daisy chaining simple coordinate
+   operations.
+
+ o Introduced cct, the Coordinate Conversion and Transformation
+   application.
+
+ o Introduced gie, the Geospatial Integrity Investigation Environment.
+   - Selftest invoked by -C flag in proj has been removed
+
+ o Adopted terminology from the OGC/ISO-19100 geospatial standards
+   series. Key definitions are:
+   - A coordinate operation is a change of coordinates, based on a
+     one-to-one relationship, from one coordinate reference system
+     to another
+   - A transformation is a coordinate operation in which the two
+     coordinate reference systems are based on different datums, e.g.
+     a change from a global reference frame to a regional frame.
+   - A coordinate operation is an operation in which both coordinate
+     reference systems are based on the same datum, e.g. change of
+     units of coordinates.
+   - A projection is a coordinate conversion from an ellipsoidal
+     coordinate system to a plane. Although projections are simply
+     conversions according to the standard, they are treated as
+     separate entities in PROJ as they make up the vast majority
+     of operations in the library.
+
+ o New operations:
+   - The pipeline operator (pipeline)
+   - Transformations:
+     + Helmert transform (helmert)
+     + Horner complex polynomial evaluation (horner)
+     + Horizontal gridshift (hgridshift)
+     + Vertical gridshift (vgridshift)
+     + Molodensky transform (molodensky)
+     + Kinematic gridshift with deformation model (deformation)
+   - Conversions:
+     + Unit conversion (unitconvert)
+     + Axis swap (axisswap)
+   - Projections:
+     + Central Conic projection (ccon)
+
+ o Significant documentation updates, including
+   - Overhaul of the structure of the documentation
+   - A better introduction to the use of PROJ
+   - A complete reference to the new proj.h API
+   - a complete rewrite of the section on geodesic calculations
+   - Figures for all projections
+
+ o New "free format" option for operation definitions, which
+   permits separating tokens by whitespace when specifying key/value-
+   pairs, e.g. "proj = merc lat_0 = 45".
+
+ o Added metadata to init-files that can be read with the
+   proj_init_info() function in the new proj.h API.
+
+ o Added ITRF2000, ITRF2008 and ITRF2014 init-files with ITRF
+   transformation parameters, including plate motion model
+   parameters.
+
+ o Added parameters for GSK2011, PZ90 and danish (similar to existing
+   andrae which has an incorrect semimajor axis) ellipsoids.
```

Comment:
andrae ?

Reply:
Yes, that's a thing:

```
 proj -le | grep andrae
   andrae a=6377104.43     rf=300.0         Andrae 1876 (Den., Iclnd.)
```

Named after Danish geodesist Andræ, but "æ" doesn't play well non-unicode character sets. I'll change it to "andrae ellipsoid" which hopefully is less confusing.